install/udev/100.autopilot.rules

```diff
@@ -0,0 +1,2 @@
+SUBSYSTEM=="tty", ATTRS{manufacturer}=="3D Robotics", ATTRS{product}=="PX4 FMU v2.x", SYMLINK+="autopilot"
+SUBSYSTEM=="tty", ATTRS{manufacturer}=="ArduPilot", SYMLINK+="autopilot"
```

Comment:
does it work if we add it to udev inside of docker? Or must it be in the host OS